ftw/simplelayout/aliasblock/browser/templates/aliasblock.pt

```diff
@@ -15,6 +15,10 @@
     </tal:authorized>
   </tal:ref-valid>
 
+  <tal:not-authorized condition="not: view/has_view_permission">
+    <p i18n:translate="label_invalid_permission">The content is no longer accessible for you</p>
```

Comment:
Update locales or do we want one big locales updated in the end (either way is ok we just mustn't forget about it)?

Reply:
I'll update at the end, once everything is approved. Updating locales on the way is a pain in the ass.